tests/Integration/ParserTest.php

```diff
@@ -210,6 +210,28 @@ public function testIssue334()
 
         $this->assertStringContainsString('This question already has an answer here', $document->getText());
     }
+
+    /**
+     * Test that issue related pdf can now be parsed:
+     * Glyphs not in the Postscript lookup table would cause "Notice: Undefined offset"
+     *
+     * @todo Note that the "ł" in przepływu is decoded as a space character. This was already
+     * the case before the PR that caused this issue and is not currently covered by this
+     * test case. However, this issue should be addressed in the future and its fix
+     * can then be incorporated into this test by uncommenting the third assertion below.
```

Comment:
Can you move that part to the commented out code in the test itself and put it in `/** */`, like

```php
/**
 * @todo note ...
 */
// $this->assertStringContainsString(...
```
This way its more clear why that code line is commented out and we don't remove it accidentally in a future release.

Reply:
Absolutely! Done.

Reply:
Thanks!